string room
---
geometry_msgs/Point coordinate